tests/test_sequence.py

```diff
@@ -63,6 +63,15 @@ def device():
     )
 
 
+@pytest.fixture
+def mock_dev_dmm():
+    # Will be replaced by MockDevice once it has a DMM.
+    return dataclasses.replace(
+        MockDevice,
+        dmm_objects=(DMM(),),
+    )
```

Comment:
My suggestion was to avoid you replacing the `MockDevice` everywhere, but this will work too, it's just more work

Reply:
Sorry I think I am missing what your suggestion was ?

Reply:
Just what I mentioned here: https://github.com/pasqal-io/Pulser/pull/564#discussion_r1276478182

I ended up using it in `test_simulation.py`, actually.

Reply:
Okay sorry I missed that comment 😢 I decided to go for this because it seemed less work than replacing all the MockDevice by the replace (and because later it will be easier to search and replace all the `mock_dev_dmm` by `MockDevice`, whereas I think it will be harder to search for `replace(MockDevice, dmm_objects=(DMM(),))`